bindings/2.200/GeometryDash.bro

```diff
@@ -5446,14 +5457,14 @@ class EditorUI : cocos2d::CCLayer, FLAlertLayerProtocol, ColorSelectDelegate, GJ
 	// TodoReturn getXMin(int) = win 0xd7470;
 	// TodoReturn getButton(char const*, int, cocos2d::SEL_MenuHandler, cocos2d::CCMenu*);
 
-	// void onPlayback(cocos2d::CCObject* sender);
+	void onPlayback(cocos2d::CCObject* sender) = win 0xc9130;
 	void onPlaytest(cocos2d::CCObject* sender) = mac 0x10160, win 0xc9480;
-	// void onSettings(cocos2d::CCObject* sender);
-	// void onCopyState(cocos2d::CCObject* sender);
-	void onDeleteAll(cocos2d::CCObject* sender);
-	void onDuplicate(cocos2d::CCObject* sender);
-	// void onEditColor(cocos2d::CCObject* sender);
-	// void onGoToLayer(cocos2d::CCObject* sender);
+	void onSettings(cocos2d::CCObject* sender) = win 0xa5570;
+	void onCopyState(cocos2d::CCObject* sender) = win 0xcb0c0;
+	// void onDeleteAll(cocos2d::CCObject* sender);
+	// void onDuplicate(cocos2d::CCObject* sender);
```

Comment:
why comment these out :(

Reply:
This was already commented before my previous commits, tried to find them but since I don't really know how to find virtual methods I just commented them again

Reply:
these arent virtual! i believe the whole class is commented out because alk didnt want to bother moving the classes from extras, althoguh thats not needed for these simple button callbacks